tests/smoke.runner.js

```diff
@@ -27,13 +27,24 @@ async function runTests (tests) {
      */
     const testFilter = process.argv[2]
 
-    if (process.env.CI || testFilter) {
+    if (process.env.CI || (testFilter && !testFilter.startsWith('--'))) {
         // sequential
         const testsFiltered = testFilter ? tests.filter(test => test.name === testFilter) : tests
         for (let test of testsFiltered) {
             await test()
         }
     } else {
+        // run sequentially
+        if ([...process.argv].includes('--sequential')) {
```

Comment:
this is some additional way to run tests? What is the difference between passing CI=true ?

Reply:
yeah, it runs the smoke tests sequentially rather than in parallel which helps to better identify which one fails. I will add some more docs to it.

Reply:
ah actually it is the same 🙈